### DISCUSSION TITLE
Conversion to Winter?

### DISCUSSION THREAD

**Discussion by berensn** [GinoPane/oc-blogtaxonomy-plugin#100] [General]
Will you be creating a winter version of the taxonomy plugin?
I tried to do a conversion myself, with a find/replace of october/rain/rainlab references to winter/storm and it *almost* works. I get an error when going to the series page. It appears to be a malformed join on the winter.blog table and the taxonomy table. While I do know enough programming to be dangerous, I don't know enough about the Laravel framework and its semantics to debug it properly.

**Comment by berensn** [comment]
I figured out the issue. Due to the roundabout way I had to install the converted plugin, the tables that were supposed to be added to the winter_blog_posts were never added. I went back and manually added them and now its working, aside from some labels looking like "Winter.blog::lang.blog.create_category"
edit: fixed - syntax error should be "**w**inter.blog::" not "**W**inter.blog::"

**Comment by GinoPane** [comment]
Hi!
Yes, it already exists, you can find it here: https://github.com/GinoPane/wn-blogtaxonomy-plugin.
